UITests/TransferMethod/SelectTransferMethodTypeTests.swift

```diff
@@ -102,4 +105,35 @@ class SelectTransferMethodTypeTests: BaseTests {
         XCTAssertTrue(app.tables["addTransferMethodTable"].exists)
         XCTAssertTrue(app.tables.staticTexts["Account Information - United States (USD)"].exists)
     }
+
+    func testSelectTransferMethodType_verifyCountrySelectionSearch() {
+        selectTransferMethodType.tapCountry()
+
+        selectTransferMethodType.typeSearch(input: "Canada")
+        XCTAssertEqual(app.tables.cells.count, 1)
+    }
+
+    func testSelectTransferMethod_clickBankAccountOpensAddTransferMethodUi () {
+        selectTransferMethodType.selectCountry(country: "United States")
+
+        app.tables["transferMethodTableView"].staticTexts.element(matching: bankAccount).tap()
```

Comment:
Add an assertion afterwards so that we know that we are actually on the Bank Account page.  Maybe Navigation Bar Title?